app/lib/contentdm/importer.rb

```diff
@@ -18,6 +18,9 @@ def initialize(input_file, data_path, default_model)
       @doc = File.open(input_file) { |f| Nokogiri::XML(f) }
       @records = @doc.xpath("//record")
       @collection = collection
+
+      @problem_record_file_name = "#{Time.now.in_time_zone.strftime('%v')}_#{collection_name[0].split(' ').join('_')}.xml"
```

Comment:
As we discussed in demo meeting, let's add a timestamp to the file name.  Maybe something like:
`@problem_record_file_name = "#{Time.now.in_time_zone.strftime('%v-%H:%M:%S')}_#{collection_name[0].split(' ').join('_')}.xml"`